scipy/special/_round.h

```diff
@@ -6,7 +6,18 @@
 #define ROUND_H
 
 #include <math.h>
-#include "cephes/dd_idefs.h"
+
+
+/* Computes fl(a+b) and err(a+b).  */
```

Comment:
I did some research to find why the volatile was there, and apparently it is to prevent aggressive optimizations from altering the results, which may have been more of a problem back then. Now it's only an issue if someone compiles with `--ffastmath` I think.